service/k8sclient/k8client.go

```diff
@@ -803,6 +838,12 @@ func (c *K8Client) getPSMDBClusters(ctx context.Context) ([]PSMDBCluster, error)
 		if message == "" && len(conditions) > 0 {
 			message = conditions[len(conditions)-1].Message
 		}
+
+		status := make([]appStatus, 0, len(cluster.Status.Replsets))
+		for _, rs := range cluster.Status.Replsets {
+			status = append(status, appStatus{rs.Size, rs.Ready})
```

Comment:
We also have mongos and config pods, let's include them too

Reply:
Included them by creating new field cluster.Status.Mongos.